back-end/Models/User.cs

```diff
@@ -18,4 +18,8 @@ public class User
     public string Password { get; set; } = null!;
 
     public string[] Stocks { get; set; } = null!;
+
+    public string[] Amis { get; set; } = null!;
+
```

Comment:
Why not directly used a Friend class instead of a list of string for Amis and Requetes

Reply:
parce que initialement on ne veut stocker le moins d'infos possible, on a juste besoin d'afficher les informations des amis lorsqu'on les affiche,alors faut juste saver leur email, par leurs infos

Reply:
aussi, lorsqu'on va chercher un user de la BD, dans la BD on stock seulement les emails